app/modules/import-export/module.php

```diff
@@ -331,6 +335,7 @@ public function upload_kit( $file, $referrer ) {
 	 */
 	public function import_kit( string $path, array $settings, bool $split_to_chunks = false ): array {
 		$this->ensure_writing_permissions();
+		$this->ensure_DOMDocument_exists();
```

Comment:
You have a mixup of two PRs. Please take these changes out of this one.
```suggestion
```

Reply:
Done.